rust/src/write.rs

```diff
@@ -121,6 +126,7 @@ impl Default for WriteOptions {
             profile: String::new(),
             chunk_size: Some(1024 * 768),
             use_chunks: true,
+            use_buffered_chunks: false,
```

Comment:
i might name this something more related to seeking like `disable_seeking`, `append_only` maybe?

Reply:
Yeah I'm thinking that makes sense. That way it's more obvious that we're guaranteeing runtime no seeking.